### PR TITLE
OSDOCS-9316# 4.13 Correct vSphere in-tree migration info

### PR DESCRIPTION
**Pull request by lpettyjo**
Version(s): 4.13 only

4.13 changes covered in this PR https://github.com/openshift/openshift-docs/pull/71307

Issue: https://issues.redhat.com/browse/OSDOCS-9316

Link to docs preview:
https://71083--ocpdocs-pr.netlify.app/openshift-enterprise/latest/storage/container_storage_interface/persistent-storage-csi-vsphere
https://71083--ocpdocs-pr.netlify.app/openshift-enterprise/latest/storage/container_storage_interface/persistent-storage-csi-vsphere

QE review:
- [ ] QE has approved this change.

Additional information:

**PTAL**: @gcharot @gnufied

### CODE REVIEW

**Review by skopacz1 — APPROVED** [review]
LGTM, but as I said in my other review for your similar PR, I'd double check that mentioning 4.15 is acceptable here, especially if this will be merged before the 4.15 GA.

File: storage/persistent_storage/persistent-storage-vsphere.adoc [hunk 1]
```diff
@@ -26,11 +26,13 @@ requested by users.
 
 [IMPORTANT]
 ====
-{product-title} defaults to using an in-tree (non-CSI) plugin to provision vSphere storage.
+For vSphere:
 
-In future {product-title} versions, volumes provisioned using existing in-tree plugins are planned for migration to their equivalent CSI driver. CSI automatic migration should be seamless. Migration does not change how you use all existing API objects, such as persistent volumes, persistent volume claims, and storage classes. For more information about migration, see xref:../../storage/container_storage_interface/persistent-storage-csi-migration.adoc#persistent-storage-csi-migration[CSI automatic migration].
+* For new installations of {product-title} 4.13, or later, automatic migration is enabled by default. Updating to {product-title} 4.15 and later also provides automatic migration.
```
Comment (gcharot) [inline storage/persistent_storage/persistent-storage-vsphere.adoc:31]: same here

Comment (lpettyjo) [reply storage/persistent_storage/persistent-storage-vsphere.adoc:31]: Same fix as above?

File: storage/container_storage_interface/persistent-storage-csi-vsphere.adoc [hunk 2]
```diff
@@ -26,13 +26,11 @@ To create CSI-provisioned persistent volumes (PVs) that mount to vSphere storage
 ====
 For vSphere:
 
-* For new installations of {product-title} 4.13, or later, automatic migration is enabled by default.
+* For new installations of {product-title} 4.13, or later, automatic migration is enabled by default. Updating to {product-title} 4.15 and later also provides automatic migration.
```
[the remark this answers is not in the document]
Comment (gcharot) [reply storage/container_storage_interface/persistent-storage-csi-vsphere.adoc:29]: yes

[the remark this answers is not in the document]
Comment (lpettyjo) [reply storage/container_storage_interface/persistent-storage-csi-vsphere.adoc:29]: Fixed.

**Comment by gcharot** [comment]
lgtm

**Pull request merged**